content/asset-tracking/using-the-sdks.textile

```diff
@@ -0,0 +1,234 @@
+
+The following code example creates some example "resolution constraints":/asset-tracking#resolution-constraints:
+
+```[android] 
+// Prepare Resolution Constraints for an asset that will be used in the Resolution Policy
+val exampleConstraints = DefaultResolutionConstraints(
+    DefaultResolutionSet( // this constructor provides one Resolution for all states
+        Resolution(
+            accuracy = Accuracy.BALANCED,
+            desiredInterval = 1000L,
+            minimumDisplacement = 1.0
+        )
+    ),
+    proximityThreshold = DefaultProximity(spatial = 1.0),
+    batteryLevelThreshold = 10.0f,
+    lowBatteryMultiplier = 2.0f
+)
+```
+
+The next step is create a default "resolution":/asset-tracking#resolution to be used:
+
+```[android]
+// Prepare the default resolution for the Resolution Policy
+val defaultResolution = Resolution(Accuracy.BALANCED, desiredInterval = 1000L, minimumDisplacement = 1.0)
+```
+
+Once these are created you can then initialize the publisher with the constraints and default resolution, and start the publisher:
+
+```[android]
+// Initialise and Start the Publisher
+val publisher = Publisher.publishers() // get the Publisher builder in default state
+    .connection(ConnectionConfiguration(ABLY_API_KEY, CLIENT_ID)) // provide Ably configuration with credentials
+    .map(MapConfiguration(MAPBOX_ACCESS_TOKEN)) // provide Mapbox configuration with credentials
+    .androidContext(this) // provide Android runtime context
+    .resolutionPolicy(DefaultResolutionPolicyFactory(defaultResolution, this)) // provide either the default resolution policy factory or your custom implementation
+    .profile(RoutingProfile.DRIVING) // provide mode of transportation for better location enhancements
+    .start()
+```
+
+h3(#publisher-start-tracking). Start tracking
+
+You can start tracking an asset by calling the @track@ method of the publisher. You need to supply the tracking identifier of the asset to be tracked, and optionally any "resolution constraints":/asset-tracking#resolution-constraints that need to be applied.
+
+The following code example demonstrates how to start tracking an asset:
+
+```[android]
+// Start tracking an asset
+publisher.track(
+    Trackable(
+        trackingId, // provide a tracking identifier for the asset
+        constraints = exampleConstraints // provide a set of Resolution Constraints
+    )
+)
+// handle asset tracking started successfully
+...
+```
+
+h3(#publisher-stop-tracking). Stop tracking
+
+You can stop tracking a trackable (asset) that is registered with the publisher using the @remove@ method, as shown in the following code:
+
+```[android]
+publisher.remove(trackable)
+```
+
+h2(#subscribing-sdk). Using the Subscribing SDK
+
+Common operations you will need to carry out on the subscriber include:
+
+* Initialization of the subscriber.
```

Comment:
The other items in this list are in infinitive form so it might be worth changing this to "Initialize the subscriber", just for the sake of consistency.

Reply:
Actually, I've just read through it again and along with the preceding "Common operations you will need to carry out on the subscriber include:" I think this line does sound better as "Initialization". Perhaps the other items on the list should instead be changed to "_Listening_ for updates sent from the publisher" etc. I'm probably just being overly pedantic at this point though 🤷

Reply:
I've reworked slightly, yes there was something off.